src/Commands/GenericCommand.php

```diff
@@ -1,41 +1,83 @@
 <?php
-
-/*
+/**
  * This file is part of the TelegramBot package.
  *
  * (c) Avtandil Kikabidze aka LONGMAN <akalongman@gmail.com>
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
-*/
+ */
+
 namespace Longman\TelegramBot\Commands;
 
-use Longman\TelegramBot\Request;
 use Longman\TelegramBot\Command;
 use Longman\TelegramBot\Entities\Update;
+use Longman\TelegramBot\Request;
 
+/**
+ * Generic command
+ */
 class GenericCommand extends Command
 {
+    /**
+     * Name
+     *
+     * @var string
+     */
     protected $name = 'Generic';
-    protected $description = 'Handle generic commands or is executed by default when a command is not found';
+
+    /**
+     * Description
+     *
+     * @var string
+     */
+    protected $description = 'Handles generic commands or is executed by default when a command is not found';
+
+    /**
+     * Usage
+     *
+     * @var string
+     */
     protected $usage = '/';
+
+    /**
+     * Version
+     *
+     * @var string
+     */
     protected $version = '1.0.0';
+
+    /**
+     * If this command is enabled
+     *
+     * @var boolean
+     */
     protected $enabled = true;
 
+    /**
+     * Execute command
+     *
+     * @todo This can't be right, as it always returns "Command: xyz not found.. :("
```

Comment:
This command is called with recursion here: 
https://github.com/akalongman/php-telegram-bot/blob/master/src/Telegram.php#L492
After that all the commands in the dirs has been parsed.